web/src/engine/package-cache/src/keyboardRequisitioner.ts

```diff
@@ -97,6 +97,18 @@ export default class KeyboardRequisitioner {
     this.pathConfig = pathConfig;
     this.cache = new StubAndKeyboardCache(keyboardLoader);
     this.cloudQueryEngine = new CloudQueryEngine(keyboardRequester, this.pathConfig);
+
+    // Handles keymanweb.com's precached keyboard array.  There is no associated promise,
+    // so there's nothing handling the `register` call's results otherwise.
+    this.cloudQueryEngine.on('unboundregister', (registration) => {
+      try {
+        if(Array.isArray(registration)) {
+          registration.forEach((entry) => {
+            this.cache.addStub(entry);
+          });
+        }
+      } finally { }
```

Comment:
Why is this empty `finally{}` needed?

Reply:
Are naked `try { }` statements permitted in JS?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch

> The `try...catch` statement is comprised of a `try` block and either a `catch` block, a `finally` block, or both.

I want to let errors pass through.
- `keyman.register` itself is an undocumented API function.
- Since there's no lead-in Promise, type-checking all possible input variants is... less than straightforward.  We can at least be confident for the keymanweb.com use case, though.

Reply:
What is a naked `try{}` supposed to do though? It doesn't do anything, does it?

Reply:
😆 time to go home!

Reply:
Yeah, this was probably end-of-day brain.  It _does_ signal that we acknowledge the potential for errors, but I guess a comment can do that too.

Reply:
Fixed via code-suggestion in the (resolved) comment-chain below.